os/board/rtl8720e/src/rtl8720e_boot.c

```diff
@@ -333,12 +334,15 @@ void board_initialize(void)
 #ifdef CONFIG_FTL_ENABLED
 	app_ftl_init();
 #endif
+	DiagPrintf("\n\r%s: before wlan init", __func__);
 #ifdef CONFIG_AMEBALITE_WIFI
 	wlan_initialize();
 #endif
+	DiagPrintf("\n\r%s: after wlan init", __func__);
 #ifdef CONFIG_AMEBALITE_BLE
 	bt_ipc_api_init_host();
 #endif
+	DiagPrintf("\n\r%s: end of board init", __func__);
```

Comment:
Could you let us know why messages above in this file are necessary?

Reply:
Hi, it's not necessary, I will take them out